siliconcompiler/toolscripts/rhel9/install-openroad.sh

```
@@ -11,34 +11,30 @@ else
 git clone $(python3 ${src_path}/_tools.py --tool openroad --field git-url) openroad
 cd openroad
 git checkout $(python3 ${src_path}/_tools.py --tool openroad --field git-commit)
 git submodule update --init --recursive
 
-# Install missing dependencies
-sudo yum install -y bison byacc
+sudo ./etc/DependencyInstaller.sh -bazel -prefix="$BAZEL_PREFIX"
+sudo chown -R $USER:$USER $BAZEL_PREFIX
 
-deps_args=""
 if [ ! -z ${PREFIX} ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)